### ISSUE TITLE
Your username and password didn't match. Please try again.

### ISSUE THREAD

**Issue by h3ku** [ponyriders/django-amazon-price-monitor#106]
I get this message when try to login with user **user** and password **password**

**Comment by dArignac** [comment]
Hi @h3ku how do you run it? Self setup or using the docker dev images?

**Comment by h3ku** [comment]
Using the make file.
-docker-build-base
-docker-build-web
-docker-up

**Comment by dArignac** [comment]
Could please try the following:

* delete the ``pricemonitor/base`` image and the ``pricemonitor/web`` image (``docker rmi pricemonitor.base && docker rmi pricemonitor/web``)
* then build only the web image as the base image is on dockerhub and will be loaded from there: ``make docker-build-web``
* then start the whole thing ``make docker-up``
* then run ``docker logs -f pm_web_1`` (assuming that the web container has the name ``pm_web_1`` (default))
* then try to login

Please tell the result, so if it works or not.

**Comment by dArignac** [comment]
I additionally clearified this in the docs.

**Comment by h3ku** [comment]
Nothing, I try this and a new clean build but nothing.

![image](https://cloud.githubusercontent.com/assets/5603877/17607950/6f4c1086-6029-11e6-81a1-9e768ac5d30b.png)
![image](https://cloud.githubusercontent.com/assets/5603877/17607955/7bf2ea44-6029-11e6-82f5-6308fcb039ec.png)

**Comment by dArignac** [comment]
You're right, the admin user was not created. I fixed this in e0bc2dd. Ypu have to rebuild the ``pricemonitor/web`` image and should delete the ``docker/postgres`` folder beforehand. Then please start it with ``make docker-up`` and test.
